packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -210,17 +214,23 @@ class IMobileDevice {
       }
       if (result.exitCode == 255 && result.stderr != null && result.stderr.contains('Could not connect to lockdownd')) {
         if (result.stderr.contains('error code -${LockdownReturnCode.pairingDialogResponsePending.code}')) {
-          throw const IOSDeviceNotTrustedError(
+          throw const IOSDeviceLockdownError(
             'Device info unavailable. Is the device asking to "Trust This Computer?"',
             LockdownReturnCode.pairingDialogResponsePending,
           );
         }
         if (result.stderr.contains('error code -${LockdownReturnCode.invalidHostId.code}')) {
-          throw const IOSDeviceNotTrustedError(
+          throw const IOSDeviceLockdownError(
             'Device info unavailable. Device pairing "trust" may have been revoked.',
             LockdownReturnCode.invalidHostId,
           );
         }
+        if (result.stderr.contains('error code -${LockdownReturnCode.plistError.code}')) {
+          throw const IOSDeviceLockdownError(
+            'Device info unavailable. Is the device running properly?',
```

Comment:
Can we give any more specific guidance here beyond 'Is the device running properly?' Would connecting and disconnecting, or power cycling the device help? Are there setup instructions we could point to?

Looking [here](https://github.com/libimobiledevice/libimobiledevice/blob/26373b3/include/libimobiledevice/lockdown.h#L37), `LOCKDOWN_E_PLIST_ERROR` apparently has a few different causes. One cause is a problem with the plist, but it can also indicate: "the pair_record certificates are wrong" and "the device did not acknowledge the [goodbye] request". Is there any way of knowing which case is being handled here?